shared/src/libs/exceptions/config-validation-error/config-validation-error.exception.ts

```diff
@@ -0,0 +1,14 @@
+type Constructor = {
+  message: string;
+  cause?: unknown;
+};
```

Comment:
maybe let's create a separate type ErrorConstructor, and reuse it in all places where such type is now created ( I mean in every separate exception now )